addons/godot-next/inspector_plugins/behavior_collection_inspector_plugin.gd

```diff
@@ -0,0 +1,91 @@
+tool
+extends EditorInspectorPlugin
+
+##### PROPERTIES #####
+
+var class_type: ClassType
```

Comment:
Change to...

    var class_type := ClassType.new()

...and you can completely omit the `_init()` implementation.